File: Type-2/Lawnchair/app/src/main/java/com/example/lawnchairdemo/MainActivity.java
package com.example.lawnchairdemo;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        WallpaperManagerCompat wallpaperManagerCompat = new WallpaperManagerCompat(this);
//        wallpaperManagerCompat
    }
}

File: Type-2/Android-HomeAssistant/app/src/main/java/com/example/androidhome_assistantdemo/MainActivity.kt
package com.example.androidhome_assistantdemo

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    private lateinit var s:SensorWorker
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        s = SensorWorker(this)//init ?
    }
}